pkg/es/config/config.go

```diff
@@ -75,6 +75,7 @@ type Configuration struct {
 	IndexRolloverFrequencySpans    string         `mapstructure:"-"`
 	IndexRolloverFrequencyServices string         `mapstructure:"-"`
 	IndexRolloverFrequencySampling string         `mapstructure:"-"`
+	ServiceCacheTTL                time.Duration  `mapstructure:"service_cache_ttl"`
```

Comment:
I am not seeing where it's being used

Reply:
in ES Factory (`plugin/storage/es/factory.go`) : `createSpanWriter` : L283

Reply:
in ES_integration_test (`plugin/storage/integration/elasticsearch_test.go`) : `initializeESFactory` : L126 : we are using it to  pass TTL as a flag